frontend/src/pages/wifiPage/WifiPage.module.css

```diff
@@ -21,6 +21,18 @@
   padding: 10px;
 }
 
+.advancedConfigButtonContainer {
+  composes: text from '../../globalStyles/Typography.module.css';
+  margin-right: auto;
+}
+
+.advancedConfigButton {
+  color: var(--greenHighlight);
```

Comment:
I think this should probably be --green, the greenHighligh isn't used anywhere anymore and looks a bit greyed-out to me